app/client/ui/UserManager.ts

```diff
@@ -816,14 +817,19 @@ const cssMemberPublicAccess = styled(cssMemberSecondary, `
 function renderTitle(resourceType: ResourceType, resource?: Resource, personal?: boolean) {
   switch (resourceType) {
     case 'organization': {
-      if (personal) { return t('Your role for this team site'); }
+      if (personal) {
+        return t('Your role for this team site');
+      }
+
+      const gristConfig = getGristConfig();
+      const gristHomeHost = gristConfig.homeUrl ? new URL(gristConfig.homeUrl).host : '';
+      const baseDomain = gristConfig.baseDomain || gristHomeHost;
+      const orgName = (resource as Organization).name;
+      const orgDisplay = isOrgInPathOnly() ? `${baseDomain}/o/${orgName}` : `${orgName}${baseDomain}`;
+
       return [
         t('Manage members of team site'),
-        !resource ? null : cssOrgName(
-          `${(resource as Organization).name} (`,
-          cssOrgDomain(`${(resource as Organization).domain}.getgrist.com`),
```

Comment:
Previously we used `.domain` prop, and now you calculate it using `.baseDomain` config.
Can you explain why? And also I see this comment on `baseDomain`:

```js

  // Base domain for constructing new URLs, should start with "." and not include port, e.g.
  // ".getgrist.com". It should be unset for localhost operation and in single-org mode.
```

This `should be unset` looks suspicious. Are you sure it will work?